swiftgen.yml

```diff
@@ -1,11 +1,18 @@
 strings:
-  inputs: GliaWidgets/Resources/en.lproj/Localizable.strings
-  outputs:
-    - templateName: structured-swift4
-      output: GliaWidgets/L10n.swift
-      params:
-        publicAccess: true
-        enumName: L10n
+  - inputs: GliaWidgets/Resources/en.lproj/Localizable.strings
+    outputs:
+      - templatePath: swiftgen-strings.stencil
+        output: GliaWidgets/Localization.swift
+        params:
+          publicAccess: false
+          enumName: Localization
+  - inputs: GliaWidgets/Resources/en.lproj/Deprecated.strings
```

Comment:
should deprecated file still be in the `swiftgen.yml`? i supposed if it is deprecated, so no change will come to this file and we don't need to touch it.